packages/jsii-pacmak/package.json

```diff
@@ -53,6 +53,7 @@
   "devDependencies": {
     "@jsii/dotnet-runtime": "^0.0.0",
     "@jsii/java-runtime": "^0.0.0",
+    "@jsii/go-runtime": "^0.0.0",
```

Comment:
The tests for the `go` runtime are similar to those of the `python` runtime... Does this not create a dependency cycle? (`@jsii/go-runtime` -> `jsii-pacmak` -> `@jsii/go-runtime`)

Reply:
Didn't get any error to that affect. Do we have protection against cycles?

Reply:
I assume this would break if there were circular deps with actual runtime `require()` statements. Since these are dev deps that are used in various non-parallel build/test phases I think its fine?

Reply:
I mean I'm literally just curious how `learn` manages to build a topological sort in order to build things in dependency orders. As long as "it works", this is fine by me!